scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/EcdhPskClientKeyExchange.java

```diff
@@ -0,0 +1,146 @@
+/*******************************************************************************
+ * Copyright 2018 University of Rostock, Institute of Applied Microelectronics and Computer Engineering
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *    Vikram (University of Rostock)- Initial creation, adapted from ECDHClientKeyExchange
+ ******************************************************************************/
+package org.eclipse.californium.scandium.dtls;
+
+import static org.eclipse.californium.elements.util.StandardCharsets.UTF_8;
+
+import java.net.InetSocketAddress;
+import java.security.PublicKey;
+import java.security.interfaces.ECPublicKey;
+import java.security.spec.ECParameterSpec;
+import java.security.spec.ECPoint;
+import java.util.Arrays;
+
+import org.eclipse.californium.elements.util.DatagramReader;
+import org.eclipse.californium.elements.util.DatagramWriter;
+import org.eclipse.californium.elements.util.StringUtil;
+import org.eclipse.californium.scandium.dtls.cipher.ECDHECryptography;
+import org.eclipse.californium.scandium.util.ByteArrayUtils;
+
+/**
+ * class EcdhPskClientKeyExchange. <br>
+ * This class EcdhPskClientKeyExchange represents the {@link ClientKeyExchange} in ECDHE_PSK mode. See <a
```

Comment:
you do not need to spell out the class name, JavaDoc already does this for you ...